src/imp/tracer_imp.js

```diff
@@ -81,7 +80,7 @@ export default class Tracer extends opentracing.Tracer {
         this._propagators = {};
         this._propagators[this._opentracing.FORMAT_HTTP_HEADERS] = new LightStepPropagator(this);
         this._propagators[this._opentracing.FORMAT_TEXT_MAP] = new LightStepPropagator(this);
-        this._propagators[this._opentracing.FORMAT_BINARY] = new UnsupportedPropagator(this,
+        this._propagators[this._opentracing.FORMAT_BINARY] = new LightStepPropagator(this,
```

Comment:
Does it make sense to have the LightstepPropagator now have a default for FORMAT_BINARY?

Reply:
It looks like go uses the `BinaryPropagator` by default: https://github.com/lightstep/lightstep-tracer-go/blob/master/tracer.go#L180. I think that corresponds to the EnvoyPropagator from this PR.

I think this makes sense, but have we thought about how this will work with or impact the no-protobuf version of this library?

Reply:
FORMAT_BINARY should correspond to the binary propagator. BinaryPropagator/EnvoyPropagator/whatever are all the same. 

Theoretically this shouldn't impact the no-pb version (as FORMAT_BINARY should map to UnsupportedPropagator correctly over there)